### ISSUE TITLE
Beta code not updating and list of installed not accurate

### ISSUE THREAD

**Issue by PrayerfulDrop** [dcmeglio/hubitat-packagemanager#108]
Beta versions are not updating based on the betaVersion number.  NOAA Weather Alerts Beta has had two updates and HPM is not catching this.

Also, in the list of apps/drivers installed if the beta version is installed then the version should be the beta version installed and not the production version.

**Comment by neonturbo** [comment]
I just updated to your latest beta. I got 3.0.003 by doing an update through HPM. I verified this is what I actually have in my Apps Code tab in Hubitat. So that part is working at least. Similar for the driver.

The _reported_ version of the app is only 2.5.4 in the "View Apps and Drivers" section of HPM. I assume this is what you are referring to? There is only 1 app and driver installed, so I cannot see any reason it should have this older version listed and not the newer.

It doesn't seem to affect anything, other than the version discrepancy.
